EncoreTickets.SDK.Tests/Helpers/ApiServiceMockers/MockersForApiService.cs

```diff
@@ -73,5 +82,22 @@ private Mock<ApiRestClientBuilder> GetApiRestClientBuilderMock(Mock<RestClientWr
                 .Returns(() => restClientWrapperMock.Object);
             return restClientBuilderMock;
         }
+
+        private bool IsJsonBodyInRequest(IRestRequest request, string bodyInJson)
+        {
+            var bodyParameter = request.Parameters.FirstOrDefault(p => p.Type == ParameterType.RequestBody);
+            if (bodyInJson == null)
+            {
+                return bodyParameter == null;
+            }
+
+            if (bodyParameter == null)
+            {
+                return false;
+            }
+
+            var serializedBody = request.JsonSerializer.Serialize(bodyParameter.Value);
```

Comment:
Is there a reason why we can't write ```var serializedBody = request.JsonSerializer.Serialize(bodyParameter.Value);``` and remove previous null checks?

Reply:
The first check is necessary to return the correct result if the request should not contain a body. The second check is needed to avoid a NullReferenceException

Reply:
Sorry, I meant ```var serializedBody = request.JsonSerializer.Serialize(bodyParameter?.Value);```. I haven't checked myself but isn't it going to return null if bodyParameter is null in which case the return statement basically would return the same result as if-statements?